data.sql

```diff
@@ -1,13 +1,22 @@
 /* Populate database with sample data. */
 
-INSERT INTO animal (name, date_of_birth, escape_attempts, neuterred, weight_kg)
+INSERT INTO animal (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Agumon', date'2020-02-03', '0', 'TRUE', '10.23');
 
-INSERT INTO animal (name, date_of_birth, escape_attempts, neuterred, weight_kg)
+INSERT INTO animal (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Gabumon', date'2018-11-15', '2', 'TRUE', '8');
 
-INSERT INTO animal (name, date_of_birth, escape_attempts, neuterred, weight_kg)
+INSERT INTO animal (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Pikachu', date'2021-01-07', '1','FALSE', '15.04');
 
-INSERT INTO animal (name, date_of_birth, escape_attempts, neuterred, weight_kg)
+INSERT INTO animal (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Devimon', date'2022-05-12', '5', 'TRUE', '11');
+
+INSERT INTO animal (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Charmander', date '2020-02-08', '0', 'TRUE','-11'),
+('Plantmon', date '2021-11-15', '2', 'TRUE', '-5.7'),
+('Squirtle', date '1993-04-02', '3', 'FALSE', '-12.13'),
+('Angemon', date '2005-06-12', '1', 'TRUE', '-45'),
+('Boarmon', date '2005-06-07', '7', 'TRUE', '20.4'),
+('Blossom', date '1998-10-13', '3', 'TRUE', '17'),
```

Comment:
- Hey, Please, let's make the value for the weight decimal since it's a floating number and not an integer here. `17.0`